ctapipe/image/tests/test_sliding_window_correction.py

```diff
@@ -0,0 +1,58 @@
+"""
+Test of sliding window extractor for LST camera pulse shape with
+the correction for the integration window completeness
+"""
+
+import numpy as np
+import astropy.units as u
+from numpy.testing import assert_allclose
+from traitlets.config.loader import Config
+
+from ctapipe.image.extractor import SlidingWindowMaxSum, ImageExtractor
+from ctapipe.image.toymodel import WaveformModel
+from ctapipe.instrument import SubarrayDescription, TelescopeDescription
+
+
+def test_sw_pulse_lst():
+    """
+    Test function of sliding window extractor for LST camera pulse shape with
+the correction for the integration window completeness
```

Comment:
wrong indentation here

Reply:
corrected, I'm not sure why precommit and flake8 did not catch this.
thx @maxnoe for reapproval
@kosack I also need it from you
and then can one of you merge the PR? I do not have permissions for doing it myself.

Reply:
Yes, will merge as soon as both approvals are there

Reply:
It is valid code and since it's a test it is probably also not checked by the documentation build